.github/workflows/reusable/setup-node.yml

```
@@ -18,32 +18,65 @@ on:
-          fetch-depth: 0
+          fetch-depth: 1              # Optimize checkout depth
 
       - name: Setup Node.js
+        id: setup-node
         uses: actions/setup-node@v4
         with:
           node-version: ${{ inputs.node-version }}
           cache: 'npm'
           cache-dependency-path: ${{ inputs.cache-dependency-path }}
+          registry-url: ${{ inputs.registry-url }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)